sphinx/texinputs/sphinx.sty

```diff
@@ -209,6 +209,61 @@
   \global\Sphinx@myfirstframedpassfalse
 }
 
+% For linebreaks inside Verbatim environment from package fancyvrb.
+\newbox\Sphinxcontinuationbox
+\newbox\Sphinxvisiblespacebox
+% These are user customizable e.g. from latex_elements's preamble key.
+% Use of \textvisiblespace for compatibility with XeTeX/LuaTeX/fontspec.
+\newcommand*\Sphinxvisiblespace
+                {\textcolor{red}{\textvisiblespace}}
```

Comment:
Oh, sorry. I understand what `Sphinxvisiblespace` means. It's OK to use `\textcolor{red}`.